includes/admin/class-wc-admin-setup-wizard.php

```diff
@@ -143,6 +144,57 @@ protected function should_show_recommended_step() {
 			|| $this->should_show_mailchimp();
 	}
 
+	public function enqueue_scripts() {
```

Comment:
Missing docblock

Reply:
A handful of other [code style issues the linter picked up, also](https://travis-ci.org/woocommerce/woocommerce/jobs/422778909#L921).